### DISCUSSION TITLE
How can I fix axes large price

### DISCUSSION THREAD

**Discussion by cazador4980** [highfestiva/finplot#306] [Q&A] [answered]
https://imgur.com/a/lTmbJyg/

If the Y axis is large on the chart,
It doesn't show normally.
What should I do to solve this problem?

3.4e+06
3.3e+06
3.2e+06
...

**Comment by highfestiva** [comment, marked as answer]
Learn [scientific notation](https://en.wikipedia.org/wiki/Scientific_notation)?

**Reply by cazador4980** [reply]
When a high number is input in the y-axis, it changes to a Scientific number.
How can I fix this as an integer?

**Reply by highfestiva** [reply]
I suppose you could change `__init__.py` (of the latest version) by making line 182 identical to line 184, but I haven't verified it. The scientific notation is coded in to the underlying graphing library, pyqtgraph, so I believe it requires a couple of lines of code to get rid of.

**Reply by cazador4980** [reply]
I solved this problem.
Thank you for giving me a clue.

If line 191 is changed as follows, it will be solved.

[format(xform(value), ',.12g') for value in values]

**Reply by highfestiva** [reply]
Good stuff. Don't know if I'll include it in the library, as you have to draw a line somewhere. For instance, ticks with 10 zeros are not really human readable, and exactly where to have that threshold is a matter of taste. I feel pyqtgraph has made a good tradeoff, so for future versions you'll have to pick between manual patching and getting accustomed to scientific notation.

**Reply by cazador4980** [reply]
I hope you will be library update with manual choise
Thank you for your reply
